packages/core/src/Select/Select.stories.tsx

```diff
@@ -0,0 +1,183 @@
+import { useState } from "react";
+import { Decorator, Meta, StoryObj } from "@storybook/react";
+
+import { HvSelect, HvSelectProps } from "./Select";
+import { HvButton } from "../Button";
+import { HvOption } from "./Option";
+import { HvOptionGroup } from "./OptionGroup";
+
+const decorator: Decorator = (Story) => (
+  <div className="w-[300px] min-h-[300px]">{Story()}</div>
+);
+
+export default {
+  title: "Components/Select",
+  component: HvSelect,
+  // @ts-expect-error https://github.com/storybookjs/storybook/issues/20782
+  subcomponents: { HvOption, HvOptionGroup },
+} satisfies Meta<typeof HvSelect>;
+
+const options = [
+  { value: "ar", label: "Argentina", flag: "🇦🇷" },
+  { value: "bg", label: "Belgium", flag: "🇧🇪" },
+  { value: "pt", label: "Portugal", flag: "🇵🇹" },
+  { value: "pl", label: "Poland", flag: "🇵🇱" },
+  { value: "sp", label: "Spain", flag: "🇪🇸" },
+  { value: "us", label: "United States", flag: "🇺🇸" },
+];
+
+export const Main: StoryObj<HvSelectProps<{}, false>> = {
+  args: {
+    multiple: false,
+  },
+  argTypes: {},
+  decorators: [decorator],
+  render: (args) => {
+    return (
+      <HvSelect
+        required
```

Comment:
When required, the border is not red if I click on the dropdown and close it without selecting anything. The error message doesn't appear also. Regarding the border, we have this for the other form elements. Was this intentional? I noticed `disabledBorder` was added to the warning component.